### PR TITLE
Add column comment for postgresql

### PR DESCRIPTION
**Pull request by kainonly**
- [x] Do only one thing
- [x] Non breaking API changes
- [x] Tested

### What did this pull request do?

Add column comment for postgresql, reimplement the `CreateTable` `AddColumn` `MigrateColumn` methods

### User Case Description

For example, define the model:

```golang
type Acl struct {
	ID   uint64
	Name string `gorm:"comment:'access control name'"`
}
```

### CODE REVIEW

**Comment by jinzhu** [comment]
Hello @kainonly 

Looks good, but for the `MigrateColumn` one, I think should check the current comment equal or not before execute?

File: migrator.go [hunk 1]
```diff
@@ -175,6 +198,24 @@ func (m Migrator) DropTable(values ...interface{}) error {
...
 
+func (m Migrator) AddColumn(value interface{}, field string) error {
+	if err := m.Migrator.AddColumn(value, field); err != nil {
+		return err
+	}
+	return m.RunWithValue(value, func(stmt *gorm.Statement) error {
+		if field := stmt.Schema.LookUpField(field); field != nil {
+			if field.Comment != "" {
+				if err := m.DB.Exec(
+					"COMMENT ON COLUMN " + stmt.Table + "." + field.DBName + " is " + field.Comment,
```
Comment (jinzhu) [inline migrator.go:209]: Hello @kainonly 

I found two issues in this line of code, which also exists on following code: 

1, `field.Comment` should be used as param
2, `COMMENT ON COLUMN ` should be written like "COMMENT ON COLUMN ?" with params `m.CurrentTable(stmt)`?

File: migrator.go [hunk 2]
```diff
@@ -192,6 +233,39 @@ func (m Migrator) HasColumn(value interface{}, field string) bool {
...
+		var description string
+		m.DB.Raw(
+			"SELECT description FROM pg_catalog.pg_description WHERE objsubid = (?) AND objoid = (?)",
+			gorm.Expr(
+				"SELECT ordinal_position FROM information_schema.columns WHERE table_name = ? AND column_name = ?",
+				stmt.Table, field.DBName,
+			),
+			gorm.Expr(
+				"SELECT oid FROM pg_catalog.pg_class WHERE relname = ? AND relnamespace = "+
+					"(SELECT oid FROM pg_catalog.pg_namespace WHERE nspname = current_schema())",
```
Comment (jinzhu) [inline migrator.go:250]: the `current_schema()` should use `m.CurrentSchema(stmt)`?

which is required if your table is defined in another namespace like `accounts.users`

**Comment by fanybook** [comment]
@kainonly @jinzhu 
Why didn't it work?

```go
type Product struct {
	gorm.Model
	Code  string	`gorm:"comment:'access control name'"`
	Price uint
}
```

**Pull request merged**